components/scream/src/physics/shoc/shoc_functions_f90.cpp

```diff
@@ -1934,25 +1959,25 @@ void shoc_diag_obklen_f(Int shcol, Real* uw_sfc, Real* vw_sfc, Real* wthl_sfc, R
     SHF::shoc_diag_obklen(uw_sfc_s, vw_sfc_s, wthl_sfc_s, wqw_sfc_s, thl_sfc_s, cldliq_sfc_s, qv_sfc_s,
                           ustar_s, kbfs_s, obklen_s);
 
-    ustar_d(i)[0] = ustar_s;
-    kbfs_d(i)[0] = kbfs_s;
-    obklen_d(i)[0] = obklen_s;
+    ustar_d(i) = ustar_s;
+    kbfs_d(i) = kbfs_s;
+    obklen_d(i) = obklen_s;
   });
 
   // Sync back to host
   std::vector<view_1d> inout_views = {ustar_d, kbfs_d, obklen_d};
-  ekat::device_to_host({ustar, kbfs, obklen}, shcol, inout_views);
+  copy_to_host({ustar, kbfs, obklen}, shcol, inout_views);
 }
 
 void shoc_pblintd_cldcheck_f(Int shcol, Int nlev, Int nlevi, Real* zi, Real* cldn, Real* pblh) {
   using SHOC    = Functions<Real, DefaultDevice>;
-  using Pack1   = typename ekat::Pack<Real, 1>;
+  using Spack   = typename SHOC::Spack;
   using Scalar  = typename SHOC::Scalar;
-  using view_2d = typename SHOC::view_2d<Pack1>;
-  using view_1d = typename SHOC::view_1d<Pack1>;
+  using view_2d = typename SHOC::view_2d<Spack>;
+  using view_1d = typename SHOC::view_1d<Scalar>;
 
-  std::vector<Int> dim1(2, shcol);
-  std::vector<Int> dim2  = {nlevi,  nlev};
+  std::vector<Int> dim1 = {shcol, shcol};
```

Comment:
This change seems to be unrelated to this PR. Was it a bug you caught?

Reply:
Yeah, I didn't mean to include that. I deleted that line (evidentially) and wrote it back slightly different. I'll change back.